pom.xml

```diff
@@ -32,6 +32,8 @@
         <solr.version>4.10.4</solr.version>
         <jena.version>2.13.0</jena.version>
         <slf4j.version>1.7.14</slf4j.version>
+        <reload4j.version>1.2.18.3</reload4j.version>
+        <slf4j-reload4j.version>1.7.15</slf4j-reload4j.version>
```

Comment:
I think this was supposed to be `1.7.35`?

Reply:
@kshepherd , should we bump of version(s)?

Reply:
@J4bbi I wasn't meaning to ask about bumping it. There is an error in the last patch that removes the hardcoded version `1.7.35` and replaces it with version `1.7.15`, with the intent of using a variable.

Reply:
good catch, got your comment mixed up with discussion above.

right, so the value here should be `1.7.35` while the value in line 1363 below should be `${slf4j-reload4j.version}`?

Reply:
i'm not averse to bumping the version as well, i've done both - used variable and bumped slf4j-reload4j and reload4j to latest stable versions. all seems to be working well.